### DISCUSSION TITLE
Has anyone successfully used snowflake-cli in Ubuntu

### DISCUSSION THREAD

**Discussion by atarijeff** [Snowflake-Labs/snowcli#1854] [Q&A]
I am trying to get snowflake-cli running in a docker Ubuntu 22.04 image for self hosted Azure Devops agent. I can get it installed from https://sfc-repo.snowflakecomputing.com/snowflake-cli/linux_x86_64/3.1.0/snowflake-cli-3.1.0.x86_64.deb but when I run the snow command it just throws Illegal Instruction. Using Python 3.10.12. Are there extra dependencies that need to be installed?

**Comment by whitford-olmst** [comment]
I think you are experiencing issue #2243.
I got this working on a GitHub Action script running `ubuntu-latest`, but had to roll back to version 3.5 (not use the latest 3.11).
The issue indicates that it will be resolved with the 3.12 release. 🤞
